engine/gfx/dynamic_pals.asm

```diff
@@ -1,20 +1,32 @@
 ClearSavedObjPals::
+	ldh a, [rSVBK]
+	push af
+	ld a, BANK(wUsedObjectPals)
+	ldh [rSVBK], a
+
 	xor a
 	ld [wUsedObjectPals], a
 	ld hl, wUsedObjectPals
 	ld bc, wNeededPalIndex - wUsedObjectPals
 	ld a, -1
 	rst ByteFill
+
+	pop af
+	ldh [rSVBK], a
 	ret
 
 DisableDynPalUpdates::
+	push hl
 	ld hl, wPalFlags
 	set DISABLE_DYN_PAL_F, [hl]
+	pop hl
 	ret
 
 EnableDynPalUpdates::
+	push hl
```

Comment:
`CheckForUsedObjPals` doesn't preserve `hl`, so either this is unnecessary or `CheckForUsedObjPals` should do it too.

Reply:
`CheckForUsedObjPals` already does. <_<